### PR TITLE
Niloofar / Bug - Fix Typescript errors in derivx

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mohammad-rework — APPROVED** [review]

File: src/pages/derivx/_start-derivx.tsx [hunk 1]
```diff
@@ -283,9 +283,11 @@ const StartDerivX = () => {
                         return (
                             <SideTab.Panel
                                 key={index}
```
Comment (eduard-deriv) [inline src/pages/derivx/_start-derivx.tsx:285]: do we really need this change? remove 3 props and transfer to an object

Comment (eduard-deriv) [reply src/pages/derivx/_start-derivx.tsx:285]: `const { label, description, item_width, mobile_item_width } = child`

use this approach to fix your problem, then you don't need to transfer the props

Comment (niloo-fs) [reply src/pages/derivx/_start-derivx.tsx:285]: Thank u 
[@Eduard](https://deriv-group.slack.com/team/U03GC37S4R4)
 for your revision, yes you are right it's better to change the props of the component,
I will change this :blush:

**Comment by eduard-deriv** [comment]
Well Done! 👍
Approved 🤗

**Pull request merged**